modin/experimental/xgboost/xgboost_ray.py

```diff
@@ -240,17 +240,17 @@ def _get_num_actors(num_actors=None):
     int
         Number of actors to create.
     """
+    if num_actors is None or isinstance(num_actors, int):
+        RuntimeError("`num_actors` must be int or None")
```

Comment:
```suggestion
    if not (num_actors is None or isinstance(num_actors, int)):
        RuntimeError("`num_actors` must be int or None")
```
looks like a bug otherwise